scripts/exglobal_fcst_nemsfv3gfs.sh

```diff
@@ -410,26 +411,32 @@ fi
 
 #-------------wavewave----------------------
 if [ $cplwav = ".true." ]; then
-#-------------wavewave----------------------
 
-  for file in $(ls $ROTDIR/${CDUMP}wave.${PDY}/${cyc}/rundata/rmp_src_to_dst_conserv_*) ; do
+  # Link WW3 files
+  for file in $(ls $COMINwave/rundata/rmp_src_to_dst_conserv_*) ; do
     $NLN $file $DATA/
   done
-  $NLN $ROTDIR/${CDUMP}wave.${PDY}/${cyc}/rundata/ww3_multi.${CDUMP}wave${WAV_MEMBER}.${cycle}.inp $DATA/ww3_multi.inp
+  $NLN $COMINwave/rundata/ww3_multi.${CDUMPwave}${WAV_MEMBER}.${cycle}.inp $DATA/ww3_multi.inp
 
+  # Check for expected wave grids for this run
```

Comment:
these kind of comments are stating the obvious fact. They should be removed.

Reply:
Removing.

Reply:
Removed.